src/storagemon/storagemon.pro

```diff
@@ -0,0 +1,50 @@
+#
+# This file is part of crash-reporter
+#
+#  Copyright (C) 2017 Jolla Ltd.
+#  Contact: Martin Kampas <martin.kampas@jolla.com>
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public License
+#  version 2.1 as published by the Free Software Foundation.
+#
+#  This program is distributed in the hope that it will be useful, but
+#  WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+#  02110-1301 USA
+#
+#
+include(../../crash-reporter-conf.pri)
+
+TEMPLATE = app
+TARGET = crash-reporter-storagemon
+
+packagesExist(qt5-boostable) {
+    DEFINES += HAS_BOOSTER
+    PKGCONFIG += qt5-boostable
+} else {
+    warning("qt5-boostable not available; startup times will be slower")
```

Comment:
Doesn't seem to actually use booster.

Reply:
How do you tell if it uses booster? Copy-pasted from daemon.pro.

Reply:
Started with invoker and gui apps using pre-initialized app and view.

Reply:
Correcting myself: qt5 booster should be compatible with inherited application class, unlike the silica/qtquick2 that have pre-instantiated QGuiApplication etc. But still if these are here, the service should use invoker for starting the binary.

Reply:
Dropped